src/Microsoft.AspNetCore.Server.IntegrationTesting.IIS/IISDeployer.cs

```diff
@@ -48,24 +51,25 @@ public override void Dispose()
                 StartTimer();
 
                 var contentRoot = string.Empty;
-                if (string.IsNullOrEmpty(DeploymentParameters.ServerConfigTemplateContent))
+                if (string.IsNullOrEmpty(IISDeploymentParameters.ServerConfigTemplateContent))
```

Comment:
All of these uses of IISDeploymentParameters will null ref if you use the wrong constructor.

Reply:
Yeah, the IISDeployerBase was incorrect by not using the copy constructor. Good catch.